### PR TITLE
CCP-1700: Added new portal position on PDP

### PR DESCRIPTION
**Pull request by patrick-scherer**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: extension-config.json [hunk 1]
```diff
@@ -1,7 +1,13 @@
 {
-  "version": "2.0.1",
+  "version": "2.0.2",
   "id": "@shopgate/upselling",
   "components": [
+    {
+      "id": "PDPDescriptionBefore",
```
Comment (aaron-martin) [inline extension-config.json]: There doesn't need to be separate configurations for PDPDescriptionBefore, PDPDescriptionAfter, and PDPHeaderAfter.  You can replace theme with one configuration where the value of target is an array like this: 
```json
{
      "id": "PDPPageSlider",
      "type": "portals",
      "path": "frontend/portals/ProductDetailPageSlider",
      "target": ["product.description.before", "product.description.after", "product.header.after"]
    },
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,13 @@
 ### Fixed
 ### Removed
 
+## [2.0.2] - 2019-09-11
```
Comment (Carsten89) [inline CHANGELOG.md]: should be 2.1.0 because it's a new feature

**Review by Carsten89 — CHANGES_REQUESTED** [review]

**Review by aaron-martin — CHANGES_REQUESTED** [review]
I only checked the portal position. That looks fine though I think you should make the improvement I suggested to the configuration. I did not check how the slider is rendered. There was an issue with the recently viewed product slider sizing. Make sure the slider renders as expected.

File: CHANGELOG.md (hunk 2 above)
Comment (patrick-scherer) [reply CHANGELOG.md]: done

File: extension-config.json (hunk 1 above)
Comment (patrick-scherer) [reply extension-config.json]: done

**Review by aaron-martin — APPROVED** [review]

**Review by Carsten89 — APPROVED** [review]

**Pull request merged**